README.md

```diff
@@ -135,8 +135,8 @@ Now add these lines to your `~/.bashrc`, `~/.profile`, or `~/.zshrc` file to hav
 
 ```sh
 export NVM_DIR="$HOME/.nvm"
-[ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"  # This loads nvm
-[ -s "$NVM_DIR/bash_completion" ] && \. "$NVM_DIR/bash_completion"  # This loads nvm bash_completion
+[ -s "$NVM_DIR/nvm.sh" ] && . "$NVM_DIR/nvm.sh"  # This loads nvm
```

Comment:
People will copy-paste this code; it's important it retains the `\`.

Reply:
But there are lines in the README, where no `\` has been placed (see section “Install script”). These lines have to be modified, too.

Reply:
ah, that's an oversight then. anything in a code block should have the `\`.